### PR TITLE
Add blog-link to error-stack README.md

### PR DESCRIPTION
**Pull request by TimDiekmann**
## 🌟 What is the purpose of this PR?

The idea is to first publish the crate, then merge #628, so the link to the blog post needs to be in the README *before* merging #628.

### CODE REVIEW

File: packages/libs/error-stack/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 Also check out our [Announcement Post] for `error-stack`!
 
-[announcement post]: https://hash.dev/###
+[announcement post]: https://hash.dev/blog/announcement-error-stack
```
Comment (Alfred-Mountfield) [inline packages/libs/error-stack/README.md:13]:
```suggestion
[announcement post]: https://hash.dev/blog/announcing-error-stack
```

Comment (nonparibus) [reply packages/libs/error-stack/README.md:13]: Came here to say this. Thank you 👼

**Review by nonparibus — CHANGES_REQUESTED** [review]
Requesting the same single change as Alfie ^^

**Review by nonparibus — APPROVED** [review]

File: packages/libs/error-stack/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 Also check out our [Announcement Post] for `error-stack`!
```
Comment (nonparibus) [inline packages/libs/error-stack/README.md:11]: 'announcement post' should not really be capitalized.

Comment (Alfred-Mountfield) [reply packages/libs/error-stack/README.md:11]: Auto-merge was a bit too trigger happy it seems. Tim has fixed #631

**Pull request merged**